packages/web/src/components/gcds-breadcrumbs/gcds-breadcrumbs-item.css

```diff
@@ -3,44 +3,49 @@
     position: relative;
     display: inline-block;
     padding: var(--gcds-breadcrumbs-item-padding) !important;
-  
+
     &:before {
       position: absolute;
       top: var(--gcds-breadcrumbs-item-arrow-top);
       left: var(--gcds-breadcrumbs-item-arrow-left);
       content: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' width='0.4rem' height='0.8rem' viewBox='0 0 8 14'><path fill='26374a' d='M7.7,6.3c0.4,0.4,0.4,1,0,1.4l-6,6c-0.4,0.4-1,0.4-1.4,0s-0.4-1,0-1.4L5.6,7L0.3,1.7c-0.4-0.4-0.4-1,0-1.4s1-0.4,1.4,0 L7.7,6.3L7.7,6.3z'/></svg>");
-      transition: filter .35s ease-in-out;
+      transition: filter 0.15s ease-in-out;
     }
-  
+
     a {
       display: inline-block;
       color: var(--gcds-breadcrumbs-default-text);
       outline: 0;
       margin: var(--gcds-breadcrumbs-item-margin);
       padding: var(--gcds-breadcrumbs-item-link-padding);
-      text-underline-position: under;
-      transition: background .35s ease-in-out, color .35s ease-in-out;
+      text-underline-offset: 0.2em;
+      text-decoration-color: currentColor;
+      text-decoration-thickness: var(--gcds-breadcrumbs-default-decoration-thickness);
+      transition: background 0.15s ease-in-out, color 0.15s ease-in-out;
       white-space: normal;
     }
   }
-  
-
-  &(:focus-within) {
-    background-color: var(--gcds-breadcrumbs-focus-background);
-    color: var(--gcds-breadcrumbs-focus-text);
 
+  &(:focus) {
     a {
-      color: inherit;
+      border-radius: var(--gcds-breadcrumbs-focus-border-radius);
+      background-color: var(--gcds-breadcrumbs-focus-background);
+      color: var(--gcds-breadcrumbs-focus-text);
+      text-decoration: none;
+      box-shadow: var(--gcds-breadcrumbs-focus-box-shadow);
+      outline-offset: var(--gcds-breadcrumbs-focus-outline-offset);
+      outline: var(--gcds-breadcrumbs-focus-outline);
     }
 
     &:before {
```

Comment:
Since the chevron is no longer being focused, we can remove the filter line.

Reply:
is `filter` what was turning the chevron white on focus?